src/util/file_util.cpp

```diff
@@ -24,25 +24,34 @@ Date: January 2012
     defined(__unix__) || \
     defined(__CYGWIN__) || \
     defined(__MACH__)
-#include <sys/stat.h>
+#include <fstream>
+#include <cstring>
+#include <cstdlib>
+#include <algorithm>
+#include <sstream>
+#include <vector>
+
 #include <unistd.h>
 #include <dirent.h>
-#include <cstdlib>
 #include <cstdio>
+#include <sys/stat.h>
 #endif
 
 #ifdef _WIN32
 #include <io.h>
+#define NOMINMAX // Don't define 'min', masking std::min
 #include <windows.h>
 #include <direct.h>
+#include <Shlwapi.h>
+#undef NOMINMAX
 #include <util/unicode.h>
 #define chdir _chdir
 #define popen _popen
 #define pclose _pclose
 #endif
 
 /// \return current working directory
-std::string get_current_working_directory()
+std::string fileutil_get_current_working_directory()
```

Comment:
In light of the recent hick-up in the value-set analysis: can we please not do arbitrary renaming? Thanks.